src/Catalog/Dnx/DnxMaker.cs

```diff
@@ -45,11 +47,12 @@ public DnxMaker(StorageFactory storageFactory)
             await DeleteNupkg(storage, id, version, cancellationToken);
         }
 
-        private async Task SaveNuspec(Storage storage, string id, string version, string nuspec, CancellationToken cancellationToken)
+        private async Task<Uri> SaveNuspec(Storage storage, string id, string version, string nuspec, CancellationToken cancellationToken)
         {
             string relativeAddress = string.Format("{1}/{0}.nuspec", id, version);
```

Comment:
Usually I save var for long names, but for you.. :)

---
In reply to: [57124149](https://github.com/NuGet/NuGet.Services.Metadata/pull/60#discussion_r57124149) [](ancestors = 57124149)